ui/position.js

```diff
@@ -142,12 +142,13 @@ $.position = {
 	getWithinInfo: function( element ) {
 		var withinElement = $( element || window ),
 			isWindow = $.isWindow( withinElement[ 0 ] ),
-			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9;
+			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9,
+			hasOffset = element && ( !isWindow && !isDocument );
 		return {
 			element: withinElement,
 			isWindow: isWindow,
 			isDocument: isDocument,
-			offset: withinElement.offset() || { left: 0, top: 0 },
+			offset: hasOffset ? $( element ).offset() : { left: 0, top: 0 },
```

Comment:
Not addressed, from Scott's comment: Why isn't this using `withinElement` anymore?